chapters/order.txt

```diff
@@ -20,6 +20,7 @@ Graphics
 	lines
 	generativemesh
 	advanced_graphics
+  textures
```

Comment:
Sure. My tab was set to 2 instead to use 4, i did not even notice it. Done in the last commit